porch/scripts/create-deployment-blueprint.sh

```diff
@@ -186,6 +186,8 @@ function customize-container-env {
 
 function main() {
```

Comment:
We probably also want to include this controller in the make targets: https://github.com/GoogleContainerTools/kpt/blob/main/porch/Makefile#L52

Reply:
Done!